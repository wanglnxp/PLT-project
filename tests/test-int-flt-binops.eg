int test(int a){
	a = 8;
	float c = 12.03;
	c = 8 + 9.9;
	print(c);
	c = a + 9.9;
	print(c);
	c = 10.9 + 8;
	print(c);
	c = 10.9 + a;
	print(c);
	return a;
}
int main(){
	test(1);
	return 0;
}