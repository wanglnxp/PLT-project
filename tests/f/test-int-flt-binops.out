17.900000
17.900000
18.900000
18.900000
